atd-vze/.env

```diff
@@ -0,0 +1,5 @@
+# IMPORTANT: YOU NEED TO INSERT *YOUR* DOMAIN AND CLIENT_ID BELOW
+REACT_APP_AUTH0_DOMAIN=atd-datatech.auth0.com
+REACT_APP_AUTH0_CLIENT_ID=Y5Q304ADgUOYq8199aSgSxLEc6Xzdkb4
+REACT_APP_AUTH0_CALLBACK_URL="https://localhost:3000/callback"
+REACT_APP_AUTH0_LOGOUT_URL=https://localhost:3000
```

Comment:
It should be safe to share these openly, we only need the id_token which is given back to us via Auth0.